troposphere/static/js/components/common/tags/Tag.react.js

```diff
@@ -25,10 +25,14 @@ export default React.createClass({
     onClick(e) {
         e.stopPropagation();
         e.preventDefault();
-        RouterInstance.getInstance()
-            .transitionTo("search",null,{ 
-                    q: this.props.tag.get('name')
-            });
+        // Still causing a warning
+        // "Failed Context Types: Required context `router`"
```

Comment:
Here's the solution:
http://stackoverflow.com/a/30241596/1213041

We need to move away from the mixins looks like and add proptypes.